editor/js/libs/ui.three.js

```diff
@@ -660,6 +673,8 @@ class UIPoints2 extends UIPoints {
 
 	setValue( points ) {
 
+		if ( this.isPointsIdentical( points ) === true ) return this;
```

Comment:
When thinking about this we should remove this bit since it's over-engineering. The overhead of generating the UI is so minor that we don't have to test for equal values.

Reply:
This identical checking here is to distinguish command path from regular path, and then decides that should point rows be re-generated or not. Blindly re-generating point rows will cause bug in regular path (see below).

Command path: The points value won't be identical there:

```
undo/execute SetGeometryCommand  # geometry is updated
   dispatch geometryChanged
      call refreshUI()
          call points.setValue(geometry.parameter.path.points # << not identical to points.getValue()
```
In this case, re-generating point rows in UIPoints is needed because UI out-synced its data.

Regular path: The points will be identical there:

```
on number input changed
  call update()
    execute SetGeometryCommand
      dispath geometryChanged
        call refreshUI()
          call points.setValue(geometry.parameter.path.points # << identical to points.getValue()
```

In this case, **re-generating point rows in UIPoints is not needed** because UI already synced its data, **otherwise, (BUG) users will slide a ghost input** (users are sliding an old input while UI is showing the new one): 

https://github.com/mrdoob/three.js/assets/1063018/fddc96a8-881d-4608-a830-e4e263a531d3

To conclude, this identical checking is needed, not just for perf, but also avoiding abovementioned bug.

Reply:
Um, let me think about this. Maybe there is a different solution that allows to implement use to implement something more simple.